src/RequestBuilder/ItemPropertiesGetRequestBuilder.php

```diff
@@ -0,0 +1,16 @@
+<?php declare(strict_types=1);
+
+namespace Lmc\Matej\RequestBuilder;
+
+use Fig\Http\Message\RequestMethodInterface;
+use Lmc\Matej\Model\Request;
+
+class ItemPropertiesGetRequestBuilder extends AbstractRequestBuilder
```

Comment:
Could you please add also simple unit test for this builder? (See other unit tests for builders).